### ISSUE TITLE
Feature request: Promptless install

### ISSUE THREAD

**Issue by beruic** [ubuntu/ubuntu-make#354]
I'm scripting some dev-setup for our company, and would like an option to accept all prompts and install without interaction.

**Comment by LyzardKing** [comment]
@didrocks This would be an "accept default destination and override if already exists". The only part that would block you would be the password prompt. What do you think?

**Comment by didrocks** [comment]
yeah, sounds like a -f for "force", the others like autoaccepting license and the rest are already options. @beruic: FYI, you can already workaround the lack of this feature (if you reinstall in an already existing path), but rm -r $DIR && umake … $DIR

**Comment by beruic** [comment]
So for example `umake ide pycharm-professional $DIR`?
This is not in the man page, og the command line help output.

**Comment by beruic** [comment]
I don't see anything for autoaccepting license either. I'm using the PPA version.

**Comment by didrocks** [comment]
Use `--help` on any framework to see options particular to that framework (not all frameworks have license, that's why the option isn't global.
So, for instance:
`umake ide pycharm-professional --help`.

PR are accepted to enhance the manpage (indeed, we should have details autogenerated for each framework and those options would appear).

**Comment by beruic** [comment]
Thank you for the information.

In order to auto generate, could you not just use the output of each commands help for that?

**Comment by didrocks** [comment]
yeah, if my memroy is correct, it's already what's happening on the man page. The missing piece is to loop over each subcommands. Would be a nice contribution! :)

**Comment by boistordu** [comment]
if we use the destdir option of pycharm for example, the shortcut in xfce menu(just an example, i don't know about the others wm) doesn't point to the good place, can we do something about that? should I open an issue?

**Comment by didrocks** [comment]
oh? definitively! Please paste the generated .desktop file from ubuntu make (path of the .desktop file and content) in a new bug report.
